src/common.props

```diff
@@ -35,7 +35,7 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
   </ItemGroup>
   <PropertyGroup>
-    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <PublishRepositoryUrl>false</PublishRepositoryUrl>
```

Comment:
Hmmmm... I wonder if this will break SourceLink then. Maybe we need to hard code the RepositoryUrl instead so it can't be overridden by a personal fork?